app/code/Magento/CatalogGraphQl/Model/Resolver/Products/Query/Search.php

```diff
@@ -141,4 +150,26 @@ private function buildSearchCriteria(array $args, ResolveInfo $info): SearchCrit
 
         return $searchCriteria;
     }
+
+
+    /**
+     * Get product attribute value
+     *
+     * @param Product $product
+     * @param $productArray
+     * @param $field
+     *
+     * @return string|null
+     */
+    private function getAttributeValue(Product $product, $productArray, $field)
```

Comment:
Please, use return type hinting `?string`

Reply:
Hi @rogyar 
this changes make some API functional test failed , for example`testMediaGalleryForProductVideos` `dev/tests/api-functional/testsuite/Magento/GraphQl/Catalog/MediaGalleryTest.php`